flask_rebar/rebar.py

```diff
@@ -146,11 +146,19 @@ def wrapped(*args, **kwargs):
         if not response_body_schema:
             return rv
 
-        data, status_code, headers = _unpack_view_func_return_value(rv)
+        if isinstance(rv, current_app.response_class):
+            schema = response_body_schema[rv.status_code]
+            # The schema may be set to None to bypass marshaling (e.g. for 204 responses).
+            if schema is None:
+                return rv
+            # Otherwise, ensure the response body conforms to the promised schema.
+            schema.loads(rv.data)  # May raise ValidationError.
```

Comment:
Before calling `schema.loads` I think we may need to call [normalize_schema](https://github.com/plangrid/flask-rebar/blob/b163ce4f136cd2f8dd89dab7f702c56eb679fa91/flask_rebar/utils/request_utils.py#L93) or risk breaking when schema is a class and not an instance.